pkg/types/timestamp/v0.0.1/timestamp_v0_0_1_schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/timestamp/timestamp_v0_0_1_schema.json",
+    "title": "Timestamp v0.0.1 Schema",
+    "description": "Schema for RFC3161 entries",
+    "type": "object",
+    "properties": {
+        "tsr": {
+            "description": "Information about the tsr file associated with the entry",
+            "type": "object",
+            "properties" : {
+                "content": {
+                    "description": "Specifies the tsr file content inline within the document",
+                    "type": "string",
+                    "format": "byte"
+                },
```

Comment:
I would also extract and store in a field called `publicKey` the x509 certificate that verified the signature of the timestamp. It's returned from https://github.com/sassoftware/relic/blob/a47b9184570dc9f78acc224b69e812a1b3ee0eb9/lib/pkcs7/verify.go#L93

Reply:
From my understanding, all of that can be extracted from the tsr file at verification time? Is it just to make it easier on the final verifier?
I get that this is gross though:

`$ ~/src/actions-build-info$ openssl ts -reply -in test.tsr -token_out | openssl pkcs7 -inform der -print_certs | sed -n '/-----BEGIN/,/-----END/p' > chain.pem`

Reply:
I'm storing it now.